ratbagd/ratbagd-resolution.c

```diff
@@ -287,13 +252,13 @@ ratbagd_resolution_set_resolution(sd_bus *bus,
 }
 
 static int
-ratbagd_resolution_get_rate(sd_bus *bus,
-			    const char *path,
-			    const char *interface,
-			    const char *property,
-			    sd_bus_message *reply,
-			    void *userdata,
-			    sd_bus_error *error)
+ratbagd_resolution_get_report_rate(sd_bus *bus,
+				   const char *path,
+				   const char *interface,
+				   const char *property,
+				   sd_bus_message *reply,
+				   void *userdata,
+				   sd_bus_error *error)
```

Comment:
This looks like it introduces some whitespace issues? Or is that just the github UI showing it in a weird way?

Reply:
Just GitHub showing it in a weird way; I had to align the parameters again due to adding `_report`.